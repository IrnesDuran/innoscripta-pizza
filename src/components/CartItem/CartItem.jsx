import React from 'react';

const CartItem =({item: {price, title, quantity}}) => {

    return(
    <div className='flex flex-col my-3'>
            <span className='capitalize mx-1'>{title}</span>
            <span className='mx-1 text-right'>{quantity} x {price} EUR</span>
    </div>

)};

export default CartItem;
