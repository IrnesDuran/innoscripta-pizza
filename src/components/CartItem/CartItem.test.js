import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


import CartItem from './CartItem';



describe('Testing CartItem',()=>{

    it('should test CartItem snapshot',()=>{
        const mockItem = {
            item: {price:10, title:"Margherita",quantity:1}
        };

        expect(shallow(<CartItem item={mockItem}/>)).toMatchSnapshot();

    });

});

