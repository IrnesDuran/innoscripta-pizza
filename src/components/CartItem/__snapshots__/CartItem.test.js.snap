// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing CartItem should test CartItem snapshot 1`] = `ShallowWrapper {}`;
